src/initialization/InitAmrCore.cpp

```diff
@@ -88,9 +88,8 @@ namespace details
     {
         amrex::ParmParse pp_amr("amr");
 
-        // Domain index space
-        amrex::Vector<int> n_cell;
-        pp_amr.getarr("n_cell", n_cell);
+        // Domain index space - we use dummy values here, then fix later
+        amrex::Vector<int> n_cell = {AMREX_D_DECL(256,256,256)};
```

Comment:
Do these values work with any MPI size?

Reply:
They should work for any MPI size as long as this is done before `initGrids` - they won't be used for anything other than setting the Geometry objects.

Reply:
Awesome :star_struck: 

And we then overwrite the geometry objects simply? Are they used before `initGrids` for something (domain decomposition or so?).

Reply:
It looks like user-specification of `n_cell` is optional, with (256,256,256) as the current default.  Do I interpret correctly?

Reply:
Looking at this again, I do not see how this path would be ever taken. If the user does not pre-inject `amr.n_cell` into the ParmParser then the below function `one_box_per_rank` us called.